samples/image-uploader/README.md

```diff
@@ -1,5 +1,8 @@
+
 # Image Uploader
 
 Simply upload images with drag & drop in the entry editor. This UI extension will create and publish the underlying assets for you.
 
 ![](https://cldup.com/eJ68bgRXqM.gif)
+
+Icon made by [Good Ware](https://www.flaticon.com/authors/good-ware)
```

Comment:
```suggestion
Icon made by [Good Ware](https://www.flaticon.com/authors/good-ware) from www.flaticon.com
```
Taken from https://support.flaticon.com/hc/en-us/articles/207248209-How-must-I-insert-the-attribution-